docs/connect.html

```diff
@@ -98,6 +99,13 @@
     <li><code>DELETE /connectors/{name}</code> - delete a connector, halting all tasks and deleting its configuration</li>
 </ul>
 
+We also provides a REST API for getting information of connector plugins:
+
+<ul>
+    <li><code>GET /connector-plugins</code><- return a list of connector plugins installed in the Kafka Connect cluster</li>
+    <li><code>PUT /connector-plugins/{connector-type}/config/validate</code> - validate the provided configuration values against the configuration definition</li>
```

Comment:
Again, we give very limited docs here, but might want to clarify how this works a bit. You can run validation, get back per-field error info, suggested values, etc.